### ISSUE TITLE
[Onboarding] No category image used for popular list suggestions

### ISSUE THREAD

**Issue by mennenia** [artsy/eigen#1895]
I was using the popular categories image thingy for the thumbnail, which  means that when you suggest genes ("real" ones I guess, with the data coming from gravity rather than the json), I'm not using the right variable to populate the thumbnail with.

FYI, searching for categories __works fine__. (hence I'm assigning P0)

Will fix tomorrow

![simulator screen shot 6 oct 2016 16 15 07](https://cloud.githubusercontent.com/assets/373860/19158622/54f2aab8-8be1-11e6-867d-89fb59566497.png)

**Comment by mennenia** [comment]
Thanks for helping out @ash!

Issue is around line 222 in `ARPersonalizeTableViewController` (with content updated from `ARPerzonaliseViewController`):

```
NSURL *geneImageURL = self.contentDisplayMode == ARTableViewContentDisplayModePlaceholder ? gene.onboardingImageURL : gene.smallImageURL;
```

Normal search and recommendation (and the popular list from the artist step) are all done through standard gravity calls. The [popular categories](https://github.com/artsy/eigen/issues/1454) come from a specific [curated](https://github.com/artsy/eigen/issues/1868), static [json file](https://github.com/artsy/eigen/issues/1826), and hence we're grabbing the images slightly differently. [Here's the PR](https://github.com/artsy/eigen/pull/1879) where it's added in, and where you can view the json file. 

Hope that helps!

**Comment by alloy** [comment]
Fixed by #1931
